k9mail/src/main/java/com/fsck/k9/Globals.java

```diff
@@ -2,12 +2,14 @@
 
 
 import android.content.Context;
+import android.support.annotation.VisibleForTesting;
 
 
 public class Globals {
     private static Context context;
 
-    static void setContext(Context context) {
+    @VisibleForTesting
+    public static void setContext(Context context) {
```

Comment:
Instead of making this method public for everyone please add a helper class to the test directory that allows public access only for test code, i.e.

```java
public class GlobalsHelper {
    public static void setContext(Context context) {
        Globals.setContext(context);
    }
}
```